owl/EASE-SAY.owl

```diff
@@ -113,6 +244,83 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-SAY.owl#ContainmentTheory -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-SAY.owl#ContainmentTheory">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-SAY.owl#ControlTheory"/>
+        <rdfs:comment rdf:datatype="http://www.w3.org/2001/XMLSchema#string">A functional relation describing that the location of some entity, the containee, is constrained to be within some region which is the interior of some other entity, the container.
```

Comment:
Could you rephrase the beginning. It reads like "ContainmentTheory is a functional relation" -- this seems not right, or?

Reply:
change to: *A schematic theory of a functional relation ....*